src/bundle/Controller/SearchController.php

```diff
@@ -140,10 +139,11 @@ public function searchAction(Request $request): Response
                 $created = $data->getCreated();
                 $creator = $data->getCreator();
                 $query = new Query();
+                $fullText = null;
                 $criteria = [];
 
                 if (null !== $queryString) {
-                    $criteria[] = new Criterion\FullText($queryString);
+                    $fullText = new Criterion\FullText($queryString);
```

Comment:
Unless you plan on taking this into account for sorting, this can be simplified to:
```php
$query->query = new Criterion\FullText($queryString);
```

Reply:
Changed in https://github.com/ezsystems/ezplatform-admin-ui/pull/825/commits/5e8832e655b513b0f439091272df69a70f3b4eda.